.circleci/config.yml

```diff
@@ -0,0 +1,73 @@
+version: 2
+jobs:
+  npm:
+    docker:
+      - image: node:8
+    steps:
+      - checkout
+      - run:
+          name: Install NodeJS dependencies
+          command: npm install --production
+      # Ruby buildpack doesn't install NPM packages, so need to send to the deployment job so they are uploaded
+      - persist_to_workspace:
+          root: .
+          paths:
+            - node_modules
+
+  test:
+    docker:
+      - image: ruby:2
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Install Ruby dependencies
+          command: bundle install
+
+      # https://docs.codeclimate.com/docs/circle-ci-test-coverage-example#section-circleci-20
+      - run:
+          name: Install Code Climate test coverage reporter
+          command: curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+      - run:
+          name: Make Code Climate test coverage reporter executable
+          command: chmod +x ./cc-test-reporter
+      - run:
+          name: Set up Code Climate test coverage reporter
+          command: ./cc-test-reporter before-build
+
+      - run:
+          name: Run tests
+          command: bundle exec rspec
+
+      - run:
+          name: Submit test report to Code Climate
+          command: ./cc-test-reporter after-build --exit-code $?
```

Comment:
Did you verify that this works? I wouldn't assume the exit code gets passed along. If not you could join them
```
command: |
  bundle exec rspec
  ./cc-test-reporter after-build --exit-code $?
```

Or maybe it's just magic and all is good :)

Reply:
Pretty sure test coverage is only acknowledged by Code Climate on the primary branch...cool if we merge, then correct after if it doesn't work?

Reply:
Of course!